test/commands/translation/push.test.ts

```diff
@@ -232,9 +232,7 @@ describe("commands/translation/push", () => {
           const stub2 = KnockApiV1.prototype.upsertTranslation as any;
           sinon.assert.calledThrice(stub2);
 
-          const expectedArgs = {
-            translationRef: undefined,
-          };
+          const expectedArgs = {};
```

Comment:
It appears the behavior changed with args when an arg was defined but not provided — before we used to get `{arg: undefined}` vs nothing at all.